Incorrect peer dependency of Cypress under 12.6.0

### What package and version are you using?

cypress-indexeddb@2.0.0

### What browser are you using?

Chrome

### What operating system are you using?

OSX

### Actual Behavior

While `npm install`

` Could not resolve dependency:
npm WARN peer cypress@"< 12.6.0" from @this-dot/cypress-indexeddb@2.0.0
npm WARN node_modules/@this-dot/cypress-indexeddb
npm WARN   dev @this-dot/cypress-indexeddb@"^2.0.0" from the root project
npm ERR! code 127
`

### Expected Behavior

As mentioned in 2.0.0 readme
`Please note, that cypress introduced the Cypress.Commands.overwriteQuery command in [12.6.0](https://docs.cypress.io/guides/references/changelog#12-6-0), therefore, cypress versions 12.0.0 to 12.5.x are not supported by this library.`

2.0.0 version depends on Cypress.Commands.overwriteQuery which exists in Cypress >= 12.6.0 but cypress-indexeddb has a peer dependency 

`"peerDependencies": {
    "cypress": "< 12.6.0"
  }`


### Reproduction

Install Cypress 12.8.1
Install @this-dot/cypress-indexeddb@2.0.0
See `Could not resolve dependency` warning.

### Additional Information

Need update peerDependency version to 
`"peerDependencies": {
    "cypress": "> 12.6.0"
  }`

--- a/libs/cypress-indexeddb/package.json
+++ b/libs/cypress-indexeddb/package.json
@@ -20,6 +20,6 @@
     "typescript"
   ],
   "peerDependencies": {
-    "cypress": "< 12.6.0"
+    "cypress": "> 12.6.0"
   }
 }

